id/websites_tou.md

```diff
@@ -0,0 +1,112 @@
+# Ketentuan Penggunaan Situs Web & Komunikasi
+
+23 Juni 2016
+
+### Ketentuan Penggunaan
+
+
+#### 1\. Pendahuluan
+
+Harap baca ketentuan keseluruhan dokumen ini  (“Ketentuan”) dengan cermat karena menjelaskan hak dan tanggung jawab Anda saat mengunjungi situs web Mozilla (“Situs Web”), atau feed terkait, media sosial, buletin, repositori kode sumber, dan email (bersama dengan Situs Web, secara kolektif disebut sebagai “Komunikasi”). Dengan mengakses atau mendaftar untuk menerima Komunikasi, berarti Anda setuju untuk terikat oleh Ketentuan ini.
```

Comment:
There are a few double spaces in this document that I wouldn't expect for a language like Indonesian?